Maquetacion/scss/tech.scss

```diff
@@ -0,0 +1,35 @@
+.tech {
+  background-color: $green;
+  box-shadow: -10px -10px $white, -10px -10px 0px 1px $green;
+  border-radius: 5px;
```

Comment:
Remember to keep the properties ordered alphabetically. 🤓

Reply:
👍